2024-02-06 14:16:49,262 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-06 14:16:49,262 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Configure stats pid to 712257
2024-02-06 14:16:49,262 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Loading settings from /user/gent/457/vsc45787/.config/wandb/settings
2024-02-06 14:16:49,262 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Loading settings from /kyukon/home/gent/457/vsc45787/bsc/Centralized_controller_accelgor/wandb/settings
2024-02-06 14:16:49,262 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-06 14:16:49,262 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-06 14:16:49,262 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-06 14:16:49,262 INFO    MainThread:712257 [wandb_init.py:_log_setup():526] Logging user logs to /kyukon/home/gent/457/vsc45787/bsc/Centralized_controller_accelgor/wandb/run-20240206_141649-lgamd8h5/logs/debug.log
2024-02-06 14:16:49,262 INFO    MainThread:712257 [wandb_init.py:_log_setup():527] Logging internal logs to /kyukon/home/gent/457/vsc45787/bsc/Centralized_controller_accelgor/wandb/run-20240206_141649-lgamd8h5/logs/debug-internal.log
2024-02-06 14:16:49,263 INFO    MainThread:712257 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x146369af6890>
2024-02-06 14:16:49,263 INFO    MainThread:712257 [wandb_init.py:init():566] calling init triggers
2024-02-06 14:16:49,263 INFO    MainThread:712257 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'nn_architecture': [30, 128, 128, 20], 'activation_function': 'Tanh', 'num_generations': 2500, 'es_popsize': 1280}
2024-02-06 14:16:49,263 INFO    MainThread:712257 [wandb_init.py:init():591] re-initializing run, found existing run on stack: g5yg613e
2024-02-06 14:16:49,264 INFO    MainThread:712257 [wandb_run.py:_finish():1962] finishing run matthias-pex/Trial_runs_on_accelgor/g5yg613e
2024-02-06 14:16:49,264 INFO    MainThread:712257 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-02-06 14:16:49,264 INFO    MainThread:712257 [wandb_run.py:_restore():2185] restore
2024-02-06 14:16:49,264 INFO    MainThread:712257 [wandb_run.py:_restore():2191] restore done
2024-02-06 14:16:52,877 INFO    MainThread:712257 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-02-06 14:16:52,877 INFO    MainThread:712257 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-02-06 14:16:52,880 INFO    MainThread:712257 [wandb_run.py:_footer_sync_info():3816] logging synced files
2024-02-06 14:16:52,899 INFO    MainThread:712257 [wandb_init.py:init():616] starting backend
2024-02-06 14:16:52,899 INFO    MainThread:712257 [wandb_init.py:init():620] setting up manager
2024-02-06 14:16:52,899 INFO    MainThread:712257 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-02-06 14:16:52,900 INFO    MainThread:712257 [wandb_init.py:init():628] backend started and connected
2024-02-06 14:16:52,903 INFO    MainThread:712257 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-02-06 14:16:52,903 INFO    MainThread:712257 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-06 14:16:52,903 INFO    MainThread:712257 [wandb_init.py:init():720] updated telemetry
2024-02-06 14:16:52,911 INFO    MainThread:712257 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-06 14:16:54,241 INFO    MainThread:712257 [wandb_run.py:_on_init():2254] communicating current version
2024-02-06 14:16:54,284 INFO    MainThread:712257 [wandb_run.py:_on_init():2263] got version response 
2024-02-06 14:16:54,285 INFO    MainThread:712257 [wandb_init.py:init():804] starting run threads in backend
2024-02-06 14:16:54,952 INFO    MainThread:712257 [wandb_run.py:_console_start():2233] atexit reg
2024-02-06 14:16:54,952 INFO    MainThread:712257 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-02-06 14:16:54,952 INFO    MainThread:712257 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-02-06 14:16:54,953 INFO    MainThread:712257 [wandb_run.py:_redirect():2178] Redirects installed.
2024-02-06 14:16:54,953 INFO    MainThread:712257 [wandb_init.py:init():847] run started, returning control to user process
2024-02-06 14:22:15,697 INFO    MainThread:712257 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-06 14:22:15,697 INFO    MainThread:712257 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-06 14:22:38,029 INFO    MainThread:712257 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-06 14:22:55,308 INFO    MainThread:712257 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-06 14:22:55,308 INFO    MainThread:712257 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-06 14:22:55,311 INFO    MainThread:712257 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-06 14:22:55,452 INFO    MainThread:712257 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-06 14:22:55,452 INFO    MainThread:712257 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-06 14:22:55,456 INFO    MainThread:712257 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-06 14:22:55,467 INFO    MainThread:712257 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-06 14:22:55,467 INFO    MainThread:712257 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-06 14:22:55,470 INFO    MainThread:712257 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-06 14:22:55,481 INFO    MainThread:712257 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-06 14:22:55,481 INFO    MainThread:712257 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-06 14:22:55,483 INFO    MainThread:712257 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-06 14:22:55,486 INFO    MainThread:712257 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-06 14:22:55,486 INFO    MainThread:712257 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-06 14:22:55,489 INFO    MainThread:712257 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-06 14:22:55,801 INFO    MainThread:712257 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-06 14:22:55,801 INFO    MainThread:712257 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-06 14:22:55,806 INFO    MainThread:712257 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-06 14:22:55,817 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-06 14:22:55,817 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Configure stats pid to 712257
2024-02-06 14:22:55,817 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Loading settings from /user/gent/457/vsc45787/.config/wandb/settings
2024-02-06 14:22:55,817 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Loading settings from /kyukon/home/gent/457/vsc45787/bsc/Centralized_controller_accelgor/wandb/settings
2024-02-06 14:22:55,817 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-06 14:22:55,817 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-06 14:22:55,817 INFO    MainThread:712257 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-06 14:22:55,817 INFO    MainThread:712257 [wandb_init.py:_log_setup():526] Logging user logs to /kyukon/home/gent/457/vsc45787/bsc/Centralized_controller_accelgor/wandb/run-20240206_142255-brgtr0ar/logs/debug.log
2024-02-06 14:22:55,818 INFO    MainThread:712257 [wandb_init.py:_log_setup():527] Logging internal logs to /kyukon/home/gent/457/vsc45787/bsc/Centralized_controller_accelgor/wandb/run-20240206_142255-brgtr0ar/logs/debug-internal.log
2024-02-06 14:22:55,818 INFO    MainThread:712257 [wandb_init.py:init():566] calling init triggers
2024-02-06 14:22:55,818 INFO    MainThread:712257 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'nn_architecture': [30, 128, 128, 20], 'activation_function': 'Tanh', 'num_generations': 2500, 'es_popsize': 6912}
2024-02-06 14:22:55,818 INFO    MainThread:712257 [wandb_init.py:init():591] re-initializing run, found existing run on stack: lgamd8h5
2024-02-06 14:22:55,819 INFO    MainThread:712257 [wandb_run.py:_finish():1962] finishing run matthias-pex/Trial_runs_on_accelgor/lgamd8h5
2024-02-06 14:22:55,819 INFO    MainThread:712257 [jupyter.py:save_history():445] not saving jupyter history
2024-02-06 14:22:55,819 INFO    MainThread:712257 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-06 14:22:55,819 INFO    MainThread:712257 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-06 14:22:55,819 INFO    MainThread:712257 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-02-06 14:22:55,819 INFO    MainThread:712257 [wandb_run.py:_restore():2185] restore
2024-02-06 14:22:55,819 INFO    MainThread:712257 [wandb_run.py:_restore():2191] restore done
2024-02-06 14:22:59,502 INFO    MainThread:712257 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-02-06 14:22:59,502 INFO    MainThread:712257 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-02-06 14:22:59,504 INFO    MainThread:712257 [wandb_run.py:_footer_sync_info():3816] logging synced files
